tests/test_scan.py

```diff
@@ -52,6 +52,13 @@
     - invalid_percentage == 0
 excluded_columns:
 - date
+sql_metrics:
+- sql: |
+    SELECT sum(size) as total_size_us
+    FROM global_temp.demodata
```

Comment:
That user should use `global_temp` is unclear IMHO. I am going to think about if we can not use a non global view.